website/spec/Mutations.md

```diff
@@ -1,9 +1,9 @@
-Relay Input Object Mutations Specification
+Relay Classic Input Object Mutations Specification
 ------------------------------------------
 
-Relay's support for mutations relies on the GraphQL server exposing
+Relay Classic's support for mutations relies on the GraphQL server exposing
 mutation fields in a standardized way. These mutations accept and emit a
-identifier string, which allows Relay to track mutations and responses.
+identifier string, which allows Relay Classic to track mutations and responses.
```

Comment:
While this was a stated intent for Modern, the identifier string (`clientMutationId`) [hasn't actually been removed _yet_](https://github.com/facebook/relay/pull/2349). I'd leave it in the upstream PR and then discuss it.